### DISCUSSION TITLE
Array of Items on a product

### DISCUSSION THREAD

**Discussion by chrisBradley-dev** [unopim/unopim#75] [Ideas]
It would be really great to have an attribute that was an array of another attribute list (maybe).

for example, if there was a list of ingredients, you could select an ingredient and then add a custom values or percentage to that line. The ingredient list would be linked to a list, and then add custom fields for the user requirement.

**Comment by devansh-pal-webkul** [comment]
@chris-chameleonbrands 
We appreciate the idea of having an attribute that is an array of multiple values or where custom values can be added with predefined values We will consider this feature and assess its feasibility for future development.
Thank you for sharing your suggestion with us!
